examples/applications/plot_prediction_latency.py

```diff
@@ -43,7 +43,7 @@ def _not_in_sphinx():
 def atomic_benchmark_estimator(estimator, X_test, verbose=False):
     """Measure runtime prediction of each instance."""
     n_instances = X_test.shape[0]
-    runtimes = np.zeros(n_instances, dtype=np.float)
+    runtimes = np.zeros(n_instances, dtype=np.int)
```

Comment:
Can you explain why it solves the problem ?
If I remember well `VisibleDeprecationWarning: using a non-integer number instead of an integer will result in an error in the future` is launched when you use a float as indices but I dont see where `runtimes` is used as indices ?